cmake/qtifw/install_win_createstartmenu.qs

```diff
@@ -47,11 +47,11 @@ function Component()
 
       component.addOperation("CreateShortcut", "@TargetDir@/Documentation/index.html", target_dir + "/EnergyPlus Documentation.lnk");
       component.addOperation("CreateShortcut", "@TargetDir@/PreProcess/EPDraw/EPDrawGUI.exe", target_dir + "/EPDrawGUI.lnk");
-      component.addOperation("CreateShortcut", "@TargetDir@/EP-Launch.exe", target_dir + "/EP-Launch-Classic.lnk");
+      component.addOperation("CreateShortcut", "@TargetDir@/EP-Launch.exe", target_dir + "/EP-Launch.lnk");
       component.addOperation("CreateShortcut", "@TargetDir@/ExampleFiles/ExampleFiles.html", target_dir + "/Example Files Summary.lnk");
       component.addOperation("CreateShortcut", "@TargetDir@/ExampleFiles/ExampleFiles-ObjectsLink.html", target_dir + "/ExampleFiles Link to Objects.lnk");
       component.addOperation("CreateShortcut", "@TargetDir@/PreProcess/IDFEditor/IDFEditor.exe", target_dir + "/IDFEditor.lnk");
-      component.addOperation("CreateShortcut", "@TargetDir@/PreProcess/IDFVersionUpdater/IDFVersionUpdater.exe", target_dir + "/IDFVersionUpdater-Classic.lnk");
+      component.addOperation("CreateShortcut", "@TargetDir@/PreProcess/IDFVersionUpdater/IDFVersionUpdater.exe", target_dir + "/IDFVersionUpdater.lnk");
```

Comment:
Remove the -Classic suffix from the legacy tools to avoid breaking anything.  This should make it an easier transition for users.